bedrock/base/templates/includes/banners/fundraiser.html

```diff
@@ -70,6 +70,6 @@
     <h2 class="c-banner-title">{{ title }}</h2>
     <p class="c-banner-tagline">{{ tagline }}</p>
 
-    <a class="c-banner-button mzp-c-button mzp-t-product mzp-t-xl" href="{{ donate_url }}" data-cta-type="button" data-cta-text="{{ button }}">{{ button }}</a>
+    <a class="c-banner-button mzp-c-button mzp-t-product mzp-t-xl" href="{{ donate_url }}" data-cta-text="Donate" data-cta-position="fundraising-banner">{{ button }}</a>
```

Comment:
Note: text values should not be translated, they should be the same no matter which locale.